### PR TITLE
solution

### PR DESCRIPTION
**Pull request by vladyslav-harkusha**
[DEMO LINK](https://vladyslav-harkusha.github.io/react_static-list-of-posts/)

### CODE REVIEW

**Review by mykola-koval — APPROVED** [review]

File: src/components/CommentList/CommentList.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.comments {
+  list-style: none;
+  &__item {
```
Comment (Elena-brv) [inline src/components/CommentList/CommentList.scss:3]:
```suggestion
  list-style: none;
  
  &__item {
```

File: src/components/UserInfo/UserInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import './UserInfo.scss';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <>
+    <p className="name">{user.name}</p>
+    <p className="email">{user.email}</p>
```
Comment (Elena-brv) [inline src/components/UserInfo/UserInfo.tsx:11]: use block `user` and classes `user__name` etc., not it's like 2 blocks

File: src/react-app-env.d.ts [hunk 3]
```diff
@@ -1 +1,22 @@
 /// <reference types="react-scripts" />
+
+interface User {
+  name: string;
+  email: string;
+}
+
+interface Commentt {
```
Comment (Elena-brv) [inline src/react-app-env.d.ts:8]:
```suggestion
interface CommentItem {
```

File: src/react-app-env.d.ts [hunk 4]
```diff
@@ -1 +1,22 @@
...
+  postId: number,
+  id: number,
+  name: string,
+  email: string,
+  body: string,
+}
+interface Post {
```
Comment (Elena-brv) [inline src/react-app-env.d.ts:15]:
```suggestion
}

interface Post {
```

**Review by Elena-brv — APPROVED** [review]

**Pull request closed without merging**